test/jdk/javax/swing/JRadioButton/4314194/bug4314194.java

```diff
@@ -40,7 +40,7 @@
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 import javax.swing.UnsupportedLookAndFeelException;
-import javax.swing.plaf.synth.SynthLookAndFeel;
+import javax.swing.plaf.nimbus.NimbusLookAndFeel;
```

Comment:
this bug id needs to be added in jtreg tag

Reply:
Updated.

Reply:
No, it's need to be updated in this test 8075917